Makefile

```diff
@@ -47,6 +47,17 @@ install/gh-release/%:
 	$* --version
 	@ echo "[$@]: Completed successfully!"
 
+install/pip/%: PYTHON ?= python
```

Comment:
can you update the ci portion of this repo with the tardigrade-ci specifics? example: https://github.com/plus3it/terraform-aws-tardigrade-config/pull/30/files didn't push it earlier b/c it was still a WIP but I think I have it mostly ironed out

Reply:
I would like to wait on this until it's been a little more fleshed out.